pkgs/applications/blockchains/clightning.nix

```
@@ -4,11 +4,11 @@
 with stdenv.lib;
 stdenv.mkDerivation rec {
   pname = "clightning";
-  version = "0.8.1";
+  version = "0.8.2";
 
   src = fetchurl {
     url = "https://github.com/ElementsProject/lightning/releases/download/v${version}/clightning-v${version}.zip";
-    sha256 = "079d3yx7yr7qrilqgaayvn18lxl8h6a1gwwbsgm5xsyxj4vdlz7r";
+    sha256 = "1w5l3r3pnhnwz3x7mjgd69cw9a18fpyjwj7kmfka7cf9hdgcwp9x";
```

```suggestion
    sha256 = "3d5dce5e83c9b1a3a6abf3482efd7528a8c45932edc97afaf8dc427b471eb4f0";
```

I've verified this hash matches the one signed with Lisa's key (who I've verified over Signal), other signatures look valid as well and match the ones in my keychain from previous versions.